docs/astronomer/amazon/aws/example_dags/example_s3/index.rst

```diff
@@ -0,0 +1,50 @@
+:py:mod:`amazon.aws.example_dags.example_s3`
+============================================
+
```

Comment:
These files are auto-generated, they shouldn't be in git!

Reply:
Removed this.